File: src/com/l2kt/gameserver/network/serverpackets/ClientSetTime.java
package com.l2kt.gameserver.network.serverpackets;

import com.l2kt.gameserver.taskmanager.GameTimeTaskManager;

public class ClientSetTime extends L2GameServerPacket
{
	@Override
	protected final void writeImpl()
	{
		writeC(0xEC);
		writeD(GameTimeTaskManager.getInstance().getGameTime());
		writeD(6);
	}
}

File: src/com/l2kt/gameserver/network/serverpackets/ClientSetTime.kt
package com.l2kt.gameserver.network.serverpackets

import com.l2kt.gameserver.taskmanager.GameTimeTaskManager

class ClientSetTime : L2GameServerPacket() {
    override fun writeImpl() {
        writeC(0xEC)
        writeD(GameTimeTaskManager.gameTime)
        writeD(6)
    }
}